src/features/amUI/common/InstanceDescription/InstanceDescription.module.css

```
@@ -0,0 +1,62 @@
+
+.metadataTerm,
+.metadataValue {
+  margin: 0;
+}
+
+.metadataTerm {
+  color: var(--ds-color-text-subtle);
+}
+
+.metadataValue {
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)